README.md

```diff
@@ -89,11 +89,14 @@ func myFunc() {
 func myFunc() {
         /*** Display the device screen size ***/
         switch Device.size() {
-            case .Screen3_5Inch: print("It's a 3.5 inch screen")
-            case .Screen4Inch:   print("It's a 4 inch screen")
-            case .Screen4_7Inch: print("It's a 4.7 inch screen")
-            case .Screen5_5Inch: print("It's a 5.5 inch screen")
-            default:             print("Unknown size")
+            case .Screen3_5Inch:  print("It's a 3.5 inch screen")
+            case .Screen4Inch:    print("It's a 4 inch screen")
```

Comment:
Here as well although this might break a bunch of peoples code now that I Think about it :)